examples/file-sharing/src/network.rs

```diff
@@ -67,8 +67,8 @@ pub(crate) async fn new(
         .kademlia
         .set_mode(Some(kad::Mode::Server));
 
-    let (command_sender, command_receiver) = mpsc::channel(0);
-    let (event_sender, event_receiver) = mpsc::channel(0);
+    let (command_sender, command_receiver) = mpsc::channel(100);
+    let (event_sender, event_receiver) = mpsc::channel(100);
```

Comment:
Leaving the buffer set to 0 allows backpressure (though I dont think tokio channels allow a 0 buffer while futures channel does)

Reply:
Yes, the tokio channels didn't work with a buffer size of 0